source/docs/articles/sites/deploybot.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Integrating Deploybot with Your Pantheon Site
+description: Learn how to set up and use Deploybot on Pantheon.
+category:
+  - developing
+keywords: deploybot, sites
+---
+
+[Deploybot](http://deploybot.com/) allows you to build and deploy code from each deployment environments/branches to one or many servers simultaneously.
+
+One of the features of Deploybot is the ability to run a build process, like Grunt, to compile CSS before deploying. See their [guide](http://deploybot.com/guides/building-assets-with-grunt-or-gulp-during-deployment) for detailed information.
```

Comment:
Explain why you'd want a build process to compile assets instead of committing them to version control. Mention DeployBot's friendly UI for managing configuration and build steps, as opposed to managing them in `yml` files like other CI service providers. Move guide link to the bottom.